python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -103,7 +112,7 @@ def eval_input_fn(batch_size, cache=False):
             ds_mapper = functools.partial(parse_sparse_feature, feature_column_names=feature_column_names, feature_metas=feature_metas)
             dataset = dataset.map(ds_mapper).batch(batch_size)
             if cache:
-                dataset = dataset.cache(filename="dataset_cache_predict.txt")
+                dataset = dataset.cache()  # TODO(shendiaomo): cache to a filename after migrated to tf2
```

Comment:
Will this change affect the performance of `pkg/sql/codegen/tensorflow`?

Reply:
Can we check the TF version and determine the cache call? Cache to memory only works on very small datasets.

Reply:
> Will this change affect the performance of `pkg/sql/codegen/tensorflow`?

No, as @typhoonzero  said, if a filename is not provided, the dataset will be cached in memory.

Reply:
> Can we check the TF version and determine the cache call? Cache to memory only works on very small datasets.

Done.